best-practices/documentation/README.md

```diff
@@ -0,0 +1,8 @@
+# Documentation
+
+## Ruby
+Ruby projects should use [Yard](http://yardoc.org/) for documentation purposes.
+
+One benefit is YARD runs a live documentation server for the community, hosting docs for all RubyGems and Github projects. The project is at [RubyDoc.info](http://www.rubydoc.info/) (formerly rdoc.info)
+
+A [useful Yardoc cheatsheet](https://gist.github.com/chetan/1827484)
```

Comment:
Should we also link to http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md?